lib/react_client/react_interop.dart

```diff
@@ -91,6 +91,16 @@ class Ref<T> {
     }
     return jsCurrent;
   }
+
+  /// Used internally to combine refs https://github.com/facebook/react/issues/13029
+  @protected
+  set current(T value) {
+    if (value is Component) {
+      jsRef.current = value.jsThis;
+    } else {
+      jsRef.current = value;
+    }
+  }
```

Comment:
Not sure if this is a big deal or not, but in `5.4.0-wip` [this setter](https://github.com/cleandart/react-dart/blob/5.4.0-wip/lib/react_client/react_interop.dart#L123-L126) isn't just used internally. Just mentioning in case there's some bad outcome of it no longer being internal when the other branch merges.

Reply:
Mm good to know. 

I think we should try and keep it `@protected` if it's not used outside of over_react, so that people don't misuse it

Reply:
We might have to follow up after this and fix usages of `.current` (assuming this merges first), since the implementation is different.